src/views/Networking.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="networking-page"
+    >
+      <input
+        v-model="room"
+        :placeholder="$t('Networking.Placeholder')"
+      >
+      <router-link :to="'/league/' + room">
+        {{ $t('Networking.Start') }}
+      </router-link>
+    </div>
+  </Page>
+</template>
+<script>
+import Page from '../components/layout/Page.vue';
+
+export default {
+  name: 'Networking',
+  components: {
+    Page,
+  },
+  data: () => ({
+    room: '',
+  }),
+};
+</script>
+<style lang="scss" scoped>
+.league-page-container {
+  padding-top: 1rem;
+}
+
+.league-page {
+  bottom: 0;
+  position: absolute;
+  top: 0;
+  width: 31rem;
+}
+
+.league-page iframe {
+  border: 0;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
```suggestion

  iframe {
    border: 0;
    height: 100%;
    width: 100%;
  }
}
```

Reply:
It can be removed because it's other page and at I remove that at conference page too because of https://github.com/aeternity/superhero-ui/blob/f6b15fc4b7f646df3c6a14e344ea438d402b44c5/src/views/Conference.vue#L23-L26

Reply:
I will change the actual height when we will start with redesign.